front-end/src/index.js

```diff
@@ -2,12 +2,19 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './App';
 import { BrowserRouter } from 'react-router-dom';
+import { Auth0Provider } from "@auth0/auth0-react";
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
+      <Auth0Provider
+    domain="dev-gdpp7l2f4g6gruhn.us.auth0.com"
```

Comment:
You should add sensitive details such as `domain` and `clientId` to `.emv` they should not be available on github.